services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDocumentBuilder.java

```diff
@@ -44,6 +50,19 @@ public static MongoDbDocumentBuilder forTenantId(final String tenant) {
         return builder.withTenantId(tenant);
     }
 
+    /**
+     * Creates a new builder for a given subject DN.
+     *
+     * @param SubjectDn The subject DN to add to the document.
+     * @return The new document builder.
+     * @throws NullPointerException if SubjectDn is {@code null}.
+     */
+    public static MongoDbDocumentBuilder forSubjectDN(final String SubjectDn) {
```

Comment:
I think we have the convention of lower camel case names for parameters.

Reply:
fixed